engines/cge2/talk.h

```diff
@@ -69,6 +69,7 @@ class Talk : public Sprite {
 public:
 	uint8 *_color;
 
+	const char *lastText = "";
```

Comment:
This does not comply with our naming conventions for class member variables (missing leading `_`).
Also this engine does not use c++11, so the initialisation of the variable should be in the constructor initialisation list and not here.
And finally, as noted in my previous comment, it seems to me you don't need to add this variable since we already have an `_oldText` variable in `InfoLine`.

Reply:
Indeed. Deleting.